### PR TITLE
Temporarily disable batch renderer training test.

### PR DESCRIPTION
**Pull request by 0mdc**
## Motivation and Context

The batch renderer training test fails, likely due to a recent sim change.

Disabling to unblock CI until the root cause is fixed.

## How Has This Been Tested

N/A

## Types of changes

- **\[Bug Fix\]** 

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes if required.

### CODE REVIEW

**Comment by 0mdc** [comment]
This may be solved by https://github.com/facebookresearch/habitat-sim/pull/2388
Let's wait for this PR to pass CI before either merging or closing.

**Review by jturner65 — APPROVED** [review]

**Pull request merged**
